Tensile/ClientWriter.py

```diff
@@ -477,6 +481,23 @@ def param(key, value):
             if convValidation:
               param('convolution-problem', problemType.convolution.identifier(problem))
 
+        if activationArgs:
+          for setting in activationArgs.settingList:
+            param('activation-enum-args', setting.activationEnum.toEnum())
+        if problemType.activationType == 'all':
+          activationCDataType = problemType.betaType if problemType.activationHPA else \
+                                problemType.dType
+          enumList = ActivationType.getEnumStrList(activationCDataType)
+          wlStr = ""
+          for actType in enumList:
+            wlStr += ActivationType(actType).toEnum() + ","
+          wlStr = wlStr.rstrip(',')
+          param('activation-whitelist', wlStr)
```

Comment:
Predicating problem should be determined by individual kernel capability.
but this list is given by caller (user).